hw/ip/pwm/dv/cov/pwm_cov.vRefine

```diff
@@ -0,0 +1,2093 @@
+    </ccf-files>
+  </information>
```

Comment:
please remove above unnecessary info like this file, so that it's easier to read
https://github.com/lowRISC/opentitan/blob/master/hw/ip/flash_ctrl/dv/cov/flash_ctrl_cov.vRefine

Reply:
https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/xcelium/cover.ccf
 file has following statement

23 // Black-box DV CDC module.
24 deselect_coverage -betfs -module prim_cdc_rand_delay

_deselct_coverage_ is supported only when coverage is invoked through ICCR.
Since PWM regression  uses IMC for coverage report, the above statment is not supported in IMC.
Supportive explaination from 'cdnshelp' is attached. 

Hence, I have commented all the coverage related to prim_cdc_rand_delay manually in **hw/ip/pwm/dv/cov/pwm_cov.vRefine**

![ICCR_and_IMC](https://user-images.githubusercontent.com/105635983/174493598-5c8727c5-0476-45d9-a1a0-985f47c3b08f.GIF)

Reply:
I'm not familiar with Xcelium coverage flow. @rasmus-madsen may know this better.

I feel we just need to add it in [cover_reg_top.ccf](https://github.com/lowRISC/opentitan/pull/13291/files#diff-42c73a7c3f91e1620a08e555ba1d844773de62d2f3034727a4f6377dc639e6b7) as well. For CSR tests, we use this ccf file as we only want to enable coverage under reg_top for those test, but the prim_cdc_rand_delay may exist under reg_top. 
I made a fix in https://github.com/lowRISC/opentitan/pull/13291 and it works for me. Please have a try